### ISSUE TITLE
PHerc.-nos as short title

### ISSUE THREAD

**Issue by HolgerEssler** [DCLP/dclpxsltbox#255]
Once #233 is in place we would like to add also the PHerc.nos as short-titles. Strictly speaking they are not "Checklist" short titles, but have the same function of referring to texts. If @paregorios, @Edelweiss, @jcowey and @rla2118 agree, we would add the relevant section to the xml-file.

**Comment by paregorios** [comment]
So, there's probably knock-on XSLT requirements here, yes?

**Comment by HolgerEssler** [comment]
As far as I can see, for PHerc. no additional XSLT is required. The only
possible problem is the slightly misleading @type="short-Checklist", but
I think it is much easier to keep this than to introduce something else
for PHerc.

**Comment by paregorios** [comment]
ok thanks

-- 
Tom Elliott, Ph.D.
Associate Director for Digital Programs and Senior Research Scholar
Institute for the Study of the Ancient World (NYU)
http://isaw.nyu.edu/people/staff/tom-elliott

**Comment by paregorios** [comment]
Blocked for resolution of #206

**Comment by paregorios** [comment]
#206 has been superseded by #212 and #269, which have added P.Herc info (when it occurs) to the heading line at the top of the page. 

@HolgerEssler @jcowey @rla2118 is this now no longer necessary?

**Comment by paregorios** [comment]
closing as OBE
